lib/pipefy_message/logging.rb

```diff
@@ -32,11 +32,11 @@ def self.logger_setup
         logger.level = LOG_LEVELS.index(ENV.fetch("ASYNC_LOG_LEVEL", "INFO")) || Logger::ERROR
 
         logger.formatter = proc do |severity, datetime, progname, msg|
-          { date: datetime.to_s,
+          { time: datetime.to_s,
```

Comment:
I think we have a DateTime here, what about changing to `datetime` instead of `time`?

Reply:
Yup, it's a datetime; I used `time` to match what we currently log on the core, at least until we have a way of delegating logging to the app's own logger ([Slack thread](https://pipefy.slack.com/archives/C03NBL5L7RQ/p1660154302953629)). Same reason I changed `app` to `program_name`. I figured maybe having the same keys (even if the sameness is due to hardcoding atm :sweat_smile:) would make queries and such easier, but I can still change it if you think it's best :blush:

Reply:
Ok, the only problem is that if we change the value of any of those keys on the core will break our gem too.
But we can proceed with this approach for now.

Reply:
:sweat_smile: Perhaps we should prioritize delegating logging to the app logger?